.github/workflows/run_tests.yml

```
@@ -0,0 +1,31 @@
+name: 'Run tests'
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+
+      - name: Install dependencies
+        run: poetry install
+
+      - name: Create DB image
+        run: docker-compose -f docker-compose-test.yml up -d
+
+      - name: Test with pytest
+        run: poetry run pytest -vv
```

Could you please add

```suggestion
        run: poetry run pytest -vvra --tb=short --cov=vitrina --cov=report=term-missing tests
```

This will make shorter and more readable trace backs, coverage and error summary report?

See also: https://github.com/atviriduomenys/spinta/blob/master/.github/workflows/test.yml#L56-L61